macros/tools/robotiq_2f_85.xacro

```diff
@@ -17,21 +17,14 @@
       <origin xyz="0 0 0" rpy="0 0 0"/>
     </xacro:robotiq_gripper>
 
+    <!-- We use grasp_link frame to perform quick tasks such as inspect surface -->
```

Comment:
This comment still applies right?
`<!-- Moveit Studio requires a link name called grasp_link to perform quick tasks such as inspect surface -->`

Reply:
I didn't like the "require" part because this link name is configurable in your Behaviors, and no longer hard-coded anywhere in the UI. So I reworded it to be more of a guideline.

Reply:
It is still required to follow our [tool coordinate convention](https://docs.picknik.ai/en/stable/getting_started/configuration_tutorials/configure_custom_robot/config_package.html?highlight=manual_grasp_link#note-coordinate-convention-for-grasp-link) to use several core behaviors in Studio.

Reply:
Yes, I think the coordinate axis convention remains true, but we're not requiring the link be necessarily named "manual_grasp_link"... since I think all Behaviors let you specify the frame name.

In the upstream PR, I changed that doc as well to express the above.

Confirming whether this is true, though?

Reply:
Customers can't see this file anyways so as long as we are good with our internal documentation (which is a different PR) then I think this is good to go.